test/Unit/Task/PhanTest.php

```diff
@@ -27,6 +27,7 @@ public function provideConfigurableOptions(): iterable
             [
                 'output_mode' => 'text',
                 'output' => null,
+                'baseline' => '',
```

Comment:
Can you add a case for baseline in `provideExternalTaskRuns` as well?

Reply:
I do not fully understand the purpose of the `provideExternalTaskRuns` tests. Can you give me a hint how I implement it properly?

Reply:
The provider is added to validate that given the task config (first argument), the command line tool is being called with the correct CLI arguments (last argument).